packages/network-tracker/package.json

```
@@ -0,0 +1,72 @@
+{
+  "name": "@streamr/network-tracker",
+  "version": "1.0.0",
+  "description": "Streamr Network Tracker package",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/streamr-dev/network-monorepo.git",
+    "directory": "packages/network-tracker"
+  },
+  "license": "STREAMR NETWORK OPEN SOURCE LICENSE",
+  "author": "Streamr Network AG <contact@streamr.network>",
```

```suggestion
  "author": "Streamr Network AG <contact@streamr.network>",
  "bin": {
    "streamr-tracker": "bin/tracker.js"
  },
```

Completely untested...